Automate/Speckle.Automate.Sdk/Runner.cs

```diff
@@ -144,3 +147,34 @@ public static async Task<int> Main<TInput>(string[] args, Func<AutomationContext
     return returnCode;
   }
 }
+
+public class SpeckleSecretProvider : JSchemaGenerationProvider
+{
+  // `GetSchema` returning `null` indicates that the given type should not have a customised schema
+  // Nullability of JSchemaTypeGenerationContext appears to be incorrect.
+#pragma warning disable CS8764 // Nullability of return type doesn't match overridden member (possibly because of nullability attributes).
```

Comment:
I've opened an issue in the Netwonsoft repo, as the nullability of `JSchemaGenerationProvider.GetSchema` appears incorrect https://github.com/JamesNK/Newtonsoft.Json.Schema/issues/341